docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,206 @@
+# Styling components
+Tizen 6.5 introduces two new methods for styling components derived from the `View` object:
+  - `ViewStyle`: designed for simple themes.
+  -  XAML file: designed for advanced themes. The styles change in runtime and can be shared between the applications.
+
+## Basic NUI themes
+The basic styling of NUI components is based on the `ViewStyle` class and its derivatives. `ViewStyle` defines attributes and stores their values. 
+
+The following is a list of the most common attributes:  
+- name
+- state 
+- sub state
+- position
+- scale
+- orientation
+- padding 
+
+You can find all the attributes and their details on [ViewStyle attributes](https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs) page.
```

Comment:
Maybe it is because it is based on Tizen 6.5, not release version. So I guess the API URL is not created... I neither don't know where to find it..

Usually in this case we insert the expected API URL, and check the link later whether it is broken or not when the platform is released.